syntax = "proto3";
package raft_log;

option cc_generic_services = true;

message SendRaftLogRequest {
  // int32 total_size = 1;
  // uint64 txn_id = 2;
  bytes raft_log = 1;
}
message SendRaftLogResponse {
  bool success = 1;
  string redirect = 2;
}

service RaftLogService {
  rpc SendRaftLog(SendRaftLogRequest) returns (SendRaftLogResponse);
};